pkg/gardenlet/controller/shoot/shoot/reconciler_reconcile.go

```diff
@@ -553,7 +553,7 @@ func (r *Reconciler) runReconcileShootFlow(ctx context.Context, o *operation.Ope
 			Fn: flow.TaskFn(func(ctx context.Context) error {
 				return botanist.DeployShootSystem(ctx)
 			}).RetryUntilTimeout(defaultInterval, defaultTimeout),
-			SkipIf:       o.Shoot.IsWorkerless || o.Shoot.HibernationEnabled,
+			SkipIf:       o.Shoot.HibernationEnabled,
```

Comment:
Now we need to move https://github.com/gardener/gardener/blob/6eb61139a0c73e3965c2a0e59eaf8ff5c88fffd5/pkg/operation/botanist/botanist.go#L188 out of the `If`.

Reply:
Thanks!